src/cascadia/TerminalApp/App.cpp

```diff
@@ -689,6 +691,22 @@ namespace winrt::TerminalApp::implementation
         _tabs[focusedTabIndex]->Scroll(delta);
     }
 
+    // Method Description:
+    // - Move the viewport of the terminal of the currently focused tab up or
+    //      down a number of lines depending on the height of the Viewpoint. 
+    //      Negative values of `delta` will move the view up, and positive values

```

Comment:
This method description looks like it needs to be updated to reflect that it's always moving one page at a time